frontend/.eslintrc

```
@@ -0,0 +1,48 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
```

```suggestion
    "sourceType": "module",
    "project": ["tsconfig.json", "tsconfig.node.json", "vite.config.ts"]
```  
    
It looks like eslint is hitting an error (https://github.com/club-40/odin-pantry/actions/runs/4854852666/jobs/8652727276?pr=25). Found a relevant issue (https://github.com/prettier/prettier-eslint/issues/201) in the `prettier-eslint` repo that suggested the above. I've tested this out locally and it should work.